client/src/controllers/TeleportController.ts

```diff
@@ -92,6 +98,22 @@ export class TeleportController extends Controller<HTMLTemplateElement> {
       throw new Error('Invalid template content.');
     }
 
+    // HACK:
+    // cloneNode doesn't run scripts, so we need to create new script elements
```

Comment:
```suggestion
    // cloneNode doesn't run scripts, so we need to create new script elements
```

This isn’t really a hack, that’s a must when implementing this kind of HTML-hydrating pattern.